README.md

```diff
@@ -521,6 +527,24 @@ See the [Development environment](#development-environment) to start the Prometh
     docker run -p 9043:9043 -e AWS_PROFILE=${AWS_PROFILE} -v $HOME/.aws:/app/.aws public.ecr.aws/qonto/prometheus-rds-exporter:latest
     ```
 
+### Istio specific steps
+
+If you use **Istio** and have **Prometheus within Istio-system**, you'll need to do the following modification before following the install instructions.
+
+1. Get your `values.yaml` for your currently deployed Prometheus system (ex: `helm get values RELEASE_NAME [flags]`)
+1. Edit the values, under `additionalScrapeConfigs` insert an additional `job_name`:
+
+    ```yaml
+        - job_name: prometheus-rds-exporter
+        kubernetes_sd_configs:
```

Comment:
I think there's a missing indent here? 🙂

Reply:
Absolutely, my copy/paste was awful, i've corrected indentation in latest commit. 🙏